securityconfig/roles.yml

```diff
@@ -192,7 +192,7 @@ cross_cluster_replication_follower_full_access:
 ml_read_access:
   reserved: true
   cluster_permissions:
-    - 'cluster:admin/openserach/ml/stats'
+    - 'cluster:admin/opensearch/ml/stats'
```

Comment:
Thanks for fixing this. 
The action name in `ml-commons` is `cluster:admin/opensearch/ml/stats/nodes`. How about changing to `cluster:admin/opensearch/ml/stats/*`

Check ml-commons code https://github.com/opensearch-project/ml-commons/blob/main/plugin/src/main/java/org/opensearch/ml/action/stats/MLStatsNodesAction.java#L13 and https://github.com/opensearch-project/ml-commons/blob/main/plugin/src/main/java/org/opensearch/ml/constant/CommonValue.java#L9

Reply:
@ylwu-amzn and I have touched base offline, @camAtGitHub could you please update the permission to be `cluster:admin/opensearch/ml/stats/nodes`, fixing two mistakes at once!